### PR TITLE
Tweak logo for dark mode

### LINKED ISSUE #11791: mne_logo.svg bottom text hard to read in dark mode
### Description of the problem

https://mne.tools/stable/_static/mne_logo.svg is not really made for dark mode. Bottom text is hard to read.

### Steps to reproduce

Look at the rendered README file on GitHub when browser set to dark mode. I used Chrome but it should not matter which one.

### Link to data

_No response_

### Expected results

Bottom text to be lighter color in dark mode.

### Actual results

Bottom text remains black, so very hard to read in dark background.

### Additional information

I happened to view the README here. I always have dark mode on.

### PR DESCRIPTION
**Pull request by cbrnr**
Fixes #11791. I made the text a bit lighter (60% gray), which makes it easier to read in dark mode. It should be still quite readable in light mode as well.

Old (current) light:
light-old

Old (current) dark:
dark-old

New light:
light

New dark:
dark

### CODE REVIEW

**Review by drammock — APPROVED** [review]
I like the choice of color, but [nitpick] the diff is quite large, because of editing via inkscape. Could you instead tweak the script that generates the logo? I think it's this line:

https://github.com/mne-tools/mne-python/blob/8ea98e2dd542735e2ae950e5daf2bdcc363f90bd/logo/generate_mne_logos.py#L129

**Comment by hoechenberger** [comment]
Looks very nice!!!

**Comment by hoechenberger** [comment]
Actually. Shouldn't we get rid of the tagline. Or change it: Brain analysis ... or something like that?

**Comment by cbrnr** [comment]
I cannot generate the updated logo:

```
findfont: Generic family 'sans-serif' not found because none of the following families were found: Primetime
/Users/clemens/Projects/mne-python/logo/generate_mne_logos.py:111: MatplotlibDeprecationWarning: The collections attribute was deprecated in Matplotlib 3.8 and will be removed two minor releases later.
  for coll in cs.collections:
findfont: Generic family 'sans-serif' not found because none of the following families were found: Cooper Hewitt
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
File ~/Projects/mne-python/logo/generate_mne_logos.py:181
    178 patch.remove()
    180 # modify to make an icon
--> 181 ax.patches.pop(-1)  # no tag line for our icon
    182 patch = Ellipse(xy, r, r, clip_on=False, zorder=-1, fc="k")
    183 ax.add_patch(patch)

AttributeError: 'ArtistList' object has no attribute 'pop'
```

**Comment by cbrnr** [comment]
And yes, I did install the font.

**Comment by hoechenberger** [comment]
Maybe you need to refresh the MPL font cache?

**Comment by cbrnr** [comment]
Thanks, that was it!

Now I saw that the logo also needs the "Primetime" font, which is only free for personal use. I'm not sure that our logo qualifies for personal use.

**Comment by cbrnr** [comment]
Replacing the Primetime font is not a good idea, the PNGs look terrible...

**Comment by cbrnr** [comment]
@larsoner please feel free to take over! Regarding your first point, I think you need to look at it with white background, because it get's hard to read because of the checkered background IMO.

Also, the SVG diffs are still massive even though they were generated with the script (but apparently, a lot has changed in how MPL renders SVGs?). But I guess that's OK.

**Comment by drammock** [comment]
> Now I saw that the logo also needs the "[Primetime](https://freefontsfamily.com/primetime-font-free-download/)" font, which is only free for personal use.

from [the designer's website](https://www.herofonts.com/):

> With a free for Personal use version, you can't use PRIMETIME © for a commercial purpose.

MNE is not commercial software, so [as I said long ago](https://github.com/mne-tools/mne-python/issues/3441#issuecomment-234424478) as long as we're not fundraising, selling the stickers, or start charging for MNE I think we're in the clear.

> @drammock okay with the switch away from PrimeTime? I think it looks fine/comparable to me but IIRC you chose the fonts originally

I chose CooperHewitt but I think @agramfort chose Primetime (and I recall being discouraged from changing it when I was scripting the logo generation because "Alex likes Primetime").  Anyway I don't think we *need* to change it, see above.

**Comment by hoechenberger** [comment]
Let's not touch Alex's Primetime then 🎉🍹

**Comment by larsoner** [comment]
Okay I think I'm happy now. @drammock @cbrnr feel free to look -- GitHub has a nice image diff nowadays! Changes:

1. Tweaked tagline the weight upward slightly -- I've noticed especially on HiDPI screens (my mac) that the `Light` font is difficult to read.
2. Added a little sanity check that matplotlib probably/hopefully finds the correct font. On my system I had several weights and it *always* used `Thin` even though I had thin/light/book/medium installed and I think it was supposed to pick Light. So I had to `rm` the other variants then `fc-cache update` and rerun. Now I have it using `Book`.
3. To offset this, I moved from black to `"0.2"` and from white to `"0.8"` which I think helps reduce the prominence close to what it was before without sacrificing readability on HiDPI displays.
4. Added `mne_logo_gray.svg` that lives in `doc/_static`. Once it deploys tonight we can change our README.rst to use `mne.tools/dev/_static/mne_logo_gray.svg`.

**Comment by larsoner** [comment]
Okay I'll actually stop pushing commits now!

File: logo/generate_mne_logos.py [hunk 1]
```diff
@@ -126,26 +132,36 @@
...
-static_dir = op.join(op.dirname(__file__), "..", "doc", "_static")
-assert op.isdir(static_dir)
-plt.savefig(op.join(static_dir, "mne_logo.svg"), transparent=True)
-tag_patch.set_facecolor("w")
-rect.set_facecolor("0.5")
-plt.savefig(op.join(static_dir, "mne_logo_dark.svg"), transparent=True)
-tag_patch.set_facecolor("k")
-rect.set_facecolor("w")
+fig.canvas.draw_idle()
+static_dir = pathlib.Path(__file__).parent / ".." / "doc" / "_static"
```
Comment (drammock) [inline logo/generate_mne_logos.py:143]: [nitpick] seems weird to mix `.parent / ".."` right after one another

```suggestion
static_dir = pathlib.Path(__file__).parent.parent / "doc" / "_static"
```

File: logo/generate_mne_logos.py [hunk 2]
```diff
@@ -126,26 +132,36 @@
...
+        transparent=True,
+    )
+
+# %%
+# mne_splash.png
 
 # modify to make the splash screen
-data_dir = op.join(op.dirname(__file__), "..", "mne", "icons")
-ax.patches[-1].set_facecolor("w")
+data_dir = pathlib.Path(__file__).parent / ".." / "mne" / "icons"
```
Comment (drammock) [inline logo/generate_mne_logos.py:162]: idem

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
@larsoner OK by me if you merge when happy

File: logo/generate_mne_logos.py (hunk 1 above)
Comment (hoechenberger) [reply logo/generate_mne_logos.py:143]: why not use `.parents[1]`

Comment (drammock) [reply logo/generate_mne_logos.py:143]: that will only work if you `resolve()`

Comment (drammock) [reply logo/generate_mne_logos.py:143]: actually never mind, `Path(__file__)` should be absolute, so the `resolve()` isn't necessary

File: doc/sphinxext/gen_commands.py [hunk 3]
```diff
@@ -51,8 +51,8 @@ def generate_commands_rst(app=None):
         from sphinx.util.display import status_iterator
     except Exception:
         from sphinx.util import status_iterator
-    root = Path(__file__).parent.parent.parent.absolute()
```
Comment (larsoner) [inline doc/sphinxext/gen_commands.py:54]: Even on this branch:
```
$ git grep "\.parent\.parent" | wc -l
70
```
but they are now at least all isolated to `tests` files, I made the others more compact in this PR

**Comment by cbrnr** [comment]
Wait, why am I seeing this on my phone in dark mode:

**Comment by larsoner** [comment]
No idea, I can't replicate on my mobile or on macOS in inspection mode as an iPhone, or in Safari

**Comment by hoechenberger** [comment]
You mean that gray bar underneath the tagline?

**Comment by hoechenberger** [comment]
Looks like this for me now in the browser and GH app:

image

image

**Comment by cbrnr** [comment]
> You mean that gray bar underneath the tagline?

No, I meant the white (gray?) text, which is 100% readable (on `main`). But apparently, this is a dark mode feature on my Android phone, because I cannot replicate on my Mac. Still, pretty neat!

**Comment by hoechenberger** [comment]
I cannot read the tagline in dark mode 🥲 (see my screenshots above)

**Comment by cbrnr** [comment]
Yes, I think it's something my phone does.

**Comment by hoechenberger** [comment]
Yeah but … wasn't that something this PR aimed to address too? 😅

**Comment by cbrnr** [comment]
Yes, but as @larsoner mentioned [here](https://github.com/mne-tools/mne-python/issues/11791#issuecomment-1800244063), we merged this prematurely. I'm not entirely sure what the plan is, but he created a third logo variant `mne_logo_gray.svg`, which I think we need to include in `README.rst`. Previously, I modified `mne_logo.svg` directly, which is the logo currently included in `README.rst`.

**Comment by cbrnr** [comment]
#12184

**Comment by hoechenberger** [comment]
Ah okay! I missed that!

**Comment by hoechenberger** [comment]
Only slightly related but I thought it was funny:

image

**Pull request merged**